### DISCUSSION TITLE
I can not login my account

### DISCUSSION THREAD

**Discussion by zoowayss** [community/community#139179] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I forget my github account email. I only remember my account name  and the password I used .
And i have 2fa code

Now  I can not login my account . How can i do ?

**Comment by zoowayss** [comment]
Who can help me

**Reply by sam-math** [reply]
try this:
Login with Username: On the GitHub login page, use your username instead of email, along with your password and 2FA code.

Check Email History: Search for old GitHub notifications in your inbox to find the associated email.
If doesn't work:
contact GitHub Support: If you’re still stuck, reach out to GitHub Support and explain the situation.

**Comment by fikri-taufiqurrahman** [comment]
If you've forgotten your GitHub account email but still remember your username, password, and have 2FA enabled, here are the steps you can try to recover your account:

- Check your password manager or email: If you use a password manager, it may have saved your email. Alternatively, search your inbox for past emails from GitHub to find which address is linked to your account.
- Try logging in with your username: Go to the GitHub login page and try logging in using your username instead of your email, along with your password and 2FA code.
- GitHub Support: If you still can't log in, you'll need to contact [GitHub Support](https://support.github.com/contact) for account recovery. Provide them with your username, and explain the situation, including the fact that you have 2FA enabled. They may ask for additional information to verify your identity.

*Make sure you have any recovery codes available if you were given them when setting up 2FA.

**Comment by ebndev** [comment]
Hi @zoowayss,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
